### ISSUE TITLE
Missing skip navigation link

### ISSUE THREAD

**Issue by iamjolly** [18F/samhsa-prototype#298]
### Problem Description
The header of the page/site offers no way to bypass repeated blocks of information, aka Skip Navigation/Skip to Main Content link. Keyboard-only and screen reader users may need to go through the repeated links each time a page loads. Providing a skip link (bypass block) helps users jump to the main content more easily, and it satisfies the [WCAG 2.4.1 - Bypass Blocks (Level A)](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-skip) requirement for Section 508 compliance.

### Solution
* [ ] Add a skip to main content link as the first focusable element in the DOM, and link it to the `main` element on the page (you'll want to give `main` an `id` attribute).
* [ ] Make the skip to main content link visually hidden when not focused.
* [ ] Show (un-hide) the element visually when it has `focus`.

### Examples/Articles
* [WebAIM - Skip Navigation Links](https://webaim.org/techniques/skipnav/)
* [Knowbility - Skip links are important](https://knowbility.org/blog/2019/skip-links/)
* [Knowbility - Skip Links Design Showcase](https://knowbility.org/blog/2019/skip-links-gallery/)
* [USWDS - Components - Header](https://designsystem.digital.gov/components/header/) - Look under "Header accessibility" for guidance about skip links.
